opensuse-base-image/src/main/docker/scripts/convert-file-based-secrets-to-props.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# Copyright 2017-2024 Open Text.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# A function for logging in the caf logging format.
+caf_log() {
+    echo "$@" |& $(dirname "$0")/../scripts/caf-log-format.sh "convert_file_based_secrets_to_props.sh" 1>&2
+}
+
+# A function for converting file-based secrets to properties.
+#
+# For example, for each environment variable ending in the _FILE suffix:
+#
+#     ABC_PASSWORD_FILE=/var/somefile.txt
+#
+# read the contents of /var/somefile.txt (for example 'mypassword'), and write the following line to /maven/secret-props.txt:
+#
+#     -DCAF.ABC_PASSWORD=mypassword
+#
+# Note that the prefix "CAF." is added to the property name.
```

Comment:
This prefix is to avoid conflicts if there happens to be a existing property with the same name